tools/src/test/resources/1.5/valid-compositions-1.5.json

```diff
@@ -55,6 +56,7 @@
       ]
     },
     {
+      "bom-ref": "composition-2",
```

Comment:
please revert this line.
we need a showcase where no (optional) `bom-ref` property is set

Reply:
Good catch. Updated test cases.